app/_components/CSVTable/CSVTable.jsx

```diff
@@ -0,0 +1,49 @@
+'use client'
+import React, { useState, useEffect } from 'react';
+import Papa from 'papaparse';
+import styles from "../../page.module.css"
+import styles_table from "./CSVTable.css"
+
+const CSVTable = () => {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    // Load the CSV file from the public folder
+    fetch('/direct_dummy_purchase_data.csv')
```

Comment:
if we were to use CSV file, we need the user to load the csv not fetching from the local